src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,17 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService user = new UserService();
+
+        if (user.findByEmail(email) != null) {
+
+            if (user.findByEmail(email).getPassword() == password) {
+                return true;
+            } else {
+                return false;
+            }
+        } else {
+            return false;
+        }
```

Comment:
I'd recommend simplifying the construction this way:
```suggestion
    return user != null && user.getPassword().equals(password);
```